### PR TITLE
Yajwin photon vision setup

### PR DESCRIPTION
**Pull request by YajwinGrover**
I am recommiting the setup for photon vision because of breach of protocol

### CODE REVIEW

**Review by mattsmithtaiko — CHANGES_REQUESTED** [review]
See comments on #17 and #18.

File: software/vision/limelight/PhotonVision.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# How PhotonVison on Limelight
```
Comment (mattsmithtaiko) [inline software/vision/limelight/PhotonVision.md:1]: Title should match filename per wiki standards (see index.md).

File: software/vision/limelight/PhotonVision.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+The goal of PhotonVision is to get the limelight to sense the AprilTags. PhotonVision is the only compatible software we found online that works with FRC. PhotonVision is essenstially OS that runs on the limelight. For more information visit their main website: [PhotonVision Main Site](https://docs.photonvision.org/en/latest/)
+
+## How to Set Up PhotonVision
+
+### Download PhotonVision
+
+[Download Steps](https://www.balena.io/etcher/)
+
+For the limelight, head on over to the [Photon Vision Gitub Page](https://github.com/photonvision/photonvision/releases). Get the latest release of Photon Vison. The name of the file you download should end with image_limelight.xy.
```
Comment (mattsmithtaiko) [inline software/vision/limelight/PhotonVision.md:13]: PhotoVision should be one word.

Comment (mattsmithtaiko) [inline software/vision/limelight/PhotonVision.md:13]: Put image_limelight.xy in code tags like this: \`image_limelight.xy\`.

File: software/vision/limelight/PhotonVisionPipeline.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Photon Vision Pipeline Process
+
+# Reflective Tape
+1. Go to: http://photonvision.local
+1. Go to the Input Tab
+    1. Change Red Balance and Blue Balance to the same values. The values should be around 20
```
Comment (mattsmithtaiko) [inline software/vision/limelight/PhotonVisionPipeline.md:6]: Add period at end.

File: software/vision/limelight/PhotonVisionPipeline.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    1. Turn down Exposure as low as you can, making sure that the tape is still visible
```
Comment (mattsmithtaiko) [inline software/vision/limelight/PhotonVisionPipeline.md:7]: Same.

File: software/vision/limelight/PhotonVisionPipeline.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+# Photon Vision Pipeline Process
+
+# Reflective Tape
+1. Go to: http://photonvision.local
+1. Go to the Input Tab
```
Comment (mattsmithtaiko) [inline software/vision/limelight/PhotonVisionPipeline.md:5]: Add period at end.

File: software/vision/limelight/PhotonVisionPipeline.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    1. Change Red Balance and Blue Balance to the same values. The values should be around 20
+    1. Turn down Exposure as low as you can, making sure that the tape is still visible
+1. Head over to the Threshold Tab
```
Comment (mattsmithtaiko) [inline software/vision/limelight/PhotonVisionPipeline.md:8]: Same.

File: software/vision/limelight/PhotonVisionPipeline.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+    1. Change the HSV sliders to encompass the color of the tape that shows up on the Raw Image.
+    1. Keep the Saturation and Hue fairly large
```
Comment (mattsmithtaiko) [inline software/vision/limelight/PhotonVisionPipeline.md:10]: Same.

**Review by mattsmithtaiko — CHANGES_REQUESTED** [review]

File: software/vision/limelight/PhotonVision.md (hunk 1 above)
Comment (YajwinGrover) [reply software/vision/limelight/PhotonVision.md:1]: Done

File: software/vision/limelight/PhotonVision.md (hunk 2 above)
Comment (YajwinGrover) [reply software/vision/limelight/PhotonVision.md:13]: Done

**Pull request merged**